src/support/global.c

```diff
@@ -47,62 +47,58 @@ __wt_endian_check(void)
 static void
 __global_calibrate_ticks(void)
 {
+	/*
+	 * Default to using __wt_epoch until we have a good value for the ratio.
+	 */
+	__wt_process.tsc_nsec_ratio = WT_TSC_DEFAULT_RATIO;
+	__wt_process.use_epochtime = true;
+
 #if defined (__i386) || defined (__amd64)
+	{
 	struct timespec start, stop;
 	double ratio;
 	uint64_t diff_nsec, diff_tsc, min_nsec, min_tsc;
 	uint64_t tries, tsc_start, tsc_stop;
 	volatile uint64_t i;
 
-	min_nsec = min_tsc = 0;
 	/*
 	 * Run this calibration loop a few times to make sure we get a
 	 * reading that does not have a potential scheduling shift in it.
 	 * The inner loop is CPU intensive but a scheduling change in the
 	 * middle could throw off calculations. Take the minimum amount
 	 * of time and compute the ratio.
 	 */
-	__wt_process.use_epochtime = false;
```

Comment:
Removing this line and setting it to true above makes all of this a moot point and we will always use wt_epoch forever.